### ISSUE TITLE
X.A.C.T. crafter

### ISSUE THREAD

**Issue by slavatyn** [GTNewHorizons/GT-New-Horizons-Modpack#13197]
### Your GTNH Discord Username

Slavatyn#0955

### Your Pack Version

2.3.0

### Mod Name

X.A.C.T.

### The Mod to Add

Good day,

https://openeye.openmods.info/mod/xact/0.5.2

We recently used this mod on a private NH server, and it really enhanced our low tier experience. It duplicates the functionality of the Forestry table, but its main feature is the ability to connect to nearby chests. The internal storage in Forestry is quite limited, but with this mod, we can place four compressed chests nearby and craft almost anything. We can't view the inventory of the nearby chests through the mod, but it retrieves items flawlessly.

Additionally, the mod has a nice feature that allows you to retrieve previous recipes. This way, you can input the final recipe, go through the components step by step, craft the initial items, and then quickly craft the final recipe through the mod by clicing back button multiply time, one step at a time.

The recipe memory is more user-friendly in the Forestry table. Perhaps there's a way to enable the Forestry bench to connect to nearby chests like in xact mod? or even more add ability go to the previous recipes (multiply but short memory until close crafter in xact)? Or maybe add the mod to the modpack?

### Your Reason

I describe it in "The Mod to Add"

### Preliminary Investigation

We added this mod to in privat server, in previous version, but I don't remember which one. In some cases, there was a bug when crafting a stack of items or using shift+craft, where the mod could produce more items than it should have. Unfortunately, I don't remember the specific situation.

This mod is more powerful than the Forestry table, so perhaps the recipe should be more challenging. As for the statement "This mod is released under an open source license. Yes, this is required now. *", I am not sure about this.

### Final Checklist

- [X] This mod is released under an open source license. Yes, this is required now.
- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this mod, please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I believe there is nothing similar in the pack already, or the existing solution isn't good enough.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members thinking it's inappropriate.
- [X] I believe this feature would make the pack better.
- [ ] I'm the official maintainer or original author of this mod for minecraft 1.7.10.

**Comment by slavatyn** [comment]
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/1012

**Comment by chochem** [comment]
judging by what you linked there its not fully compatible?

**Comment by slavatyn** [comment]
I regret that I don't know all the details. The author abandoned this mod a long time ago and released version MC 1.6.4 v0.4.3, according to this website:

https://www.minecraftforum.net/forums/mapping-and-modding-java-edition/minecraft-mods/wip-mods/1442410-xact-xhamolk_s-advanced-crafting-table-mc-1-6-4

I don't remember how I obtained version 0.5.6, which is compatible with 1.7.10, but I do remember it was challenging. The mod worked fully on NH on a private server/client, except for the rare bug I mentioned earlier. I have uploaded the file: https://drive.google.com/file/d/1Awr5lqbTCcvw7w4dgk-mcVoGOeQBzlpL/view?usp=sharing

Now, it's easy to find its mentions here: https://openeye.openmods.info/mod/xact/0.5.2
However, I still don't understand whether it's possible to download from this website or what it does.
